src/scenario.h

```diff
@@ -89,6 +100,8 @@ typedef struct {
 	sint32 flags;				// 0x0268
 	uint32 company_value;		// 0x026C
 	char completed_by[64];		// 0x0270
+	uint8 source_game;			// new in OpenRCT2
+	sint16 source_index;		// new in OpenRCT2
```

Comment:
Ah, I didn't realise I'd remove backwards compatibility. I did test saving of scores -- those work just fine, but I can see why it breaks.

Do you have any suggestions to do it differently? As I said before, I'd rather not have to keep this data in another array and have it stay sorted consistently with the scenario list so as to not impact performance...